rsts/community/index.rst

```diff
@@ -85,12 +93,27 @@ If you'd like to give a 3-5 minute ⚡ Lightning Talk ⚡ during our Open Source
     :text: Lightning Talk Sign Up Sheet
     :classes: btn-outline-secondary
 
+Office Hours
+------------
+
+Flyte maintainers are available on Zoom for you to ask anything, weekly on Wednesdays:
+
+.. link-button:: https://www.addevent.com/event/zF10349020/
+    :type: url
+    :text: 7:00 am PT / 3:00 pm GMT - Haytham Abuelfutuh
```

Comment:
let's future-proof this by removing the name, so just `7:00 am PT / 3:00 pm GMT`

Reply:
Good idea.